README.md

```diff
@@ -4,25 +4,22 @@ Thanks to everyone that is starring, forking, writing issues, pull requesting an
 
 ![updatedgif45729805790](https://user-images.githubusercontent.com/81049050/150833862-0a828939-f267-4bd2-931f-79df55d51e28.gif)
 
-# Dependecies (all Python dependecies will automatically be installed):
-1.  mpv player https://mpv.io/  (Must be installed through a package manager)
-2.  youtube-dl https://github.com/ytdl-org  (Python dependency)
+## Dependecies:
+- mpv player https://mpv.io/  (Must be installed through a package manager)
 
-------
-# How to install:
+## How to install:
 
-## To download with and install with Git:
-#### For UNIX based OS - One Line Execution
+### For UNIX based OS - One Line Execution
+Checkout the source code with `git` or download it as a .zip file.
 ```bash
-#Sudo isn't required as long as you're not in Root
-git clone https://github.com/trakBan/spongebob-cli.git
 cd spongebob-cli
-sudo chmod +x spongebob-cli
+chmod +x spongebob-cli
```

Comment:
`chmod` can be run without sudo in this situation as the current user already owns this file (unless for some reason they cloned the repository or downloaded the zip as root and then switched to a regular user)

Reply:
sudo should stay since a lot of people clone the repo.

Reply:
Yea sudo should stay.

Reply:
sorry i meant if they cloned the repository as root - my bad

if you're cloning a repository though...you have permission to chmod, no sudo needed

Reply:
i just say this because it's generally not a good idea to run commands like `git clone` that don't require root permissions with sudo - same with `chmod`. the only command here that is always required to used sudo (unless for some reason you cloned the repository or downloaded the zip file as root) is `setup.py` since that needs access to places like `/usr/bin`